library/std/src/keyword_docs.rs

```diff
@@ -651,16 +651,21 @@ mod if_keyword {}
 
 #[doc(keyword = "impl")]
 //
-/// Implement some functionality for a type.
+/// * An `impl` block is an item that is used to implement some functionality for a type.
+/// * An `impl Trait` in a type-position can be used to designate a type that implements a trait called `Trait`.
```

Comment:
Rustdoc tries the hardest to turn the first paragraph into a summary line that is used in the search results (and on module overview pages (not applicable here)).

In this case the list items are strung together without any separator (which looks like *… type.An …*). Arguably rustdoc could do better here and add an extra space.

Still, I think this should have a designated / proper summary line (for the search result page). Starting things off with a list instead of prose is generally slightly unusual independent of that.